docs/api-reference/provenance-database.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 10
+title: Provenance Database
+description: DBOS provenance database reference
+---
+
+The provenance database is an append-only versioned replica of your application database maintained by DBOS Cloud, which is the key enabler of [Interactive Time Travel](../cloud-tutorials/interactive-timetravel.md) and [Time Travel Debugging](../cloud-tutorials/timetravel-debugging.md).
+
+The provenance database can be accessed under your application database name with a `_dbos_prov` suffix. For example, if your application database name is `hello`, then your provenance database is `hello_dbos_prov`.
+
+## Application Tables
+The provenance database contains the history (within the time travel [data retention period](https://www.dbos.dev/pricing)) of each of your database tables, where you can view all versions of records in that table.
+
+To enable time travel, DBOS Cloud extends each of your tables with four additional columns:
+- `begin_xid`: The transaction ID that added the record/row. Each insert or update of the record in your application database creates a new version of the record in the provenance database. You can use this column to check which transaction created or updated this record.
+- `end_xid`: The transaction ID that deleted the record/row, or superseded the record with a new version. Each delete or update of the record in your application database does not delete the old record in the provenance database, instead, DBOS updates the `end_xid` field of the latest record (with `end_xid` set to "infinity" `9223372036854775807`) to the transaction that deleted it. You can use this column to check which transaction deleted or updated this record.
```

Comment:
It is implied but not quite explicitly stated that these are the transaction IDs from your app database.